README.md

```diff
@@ -1,25 +1,56 @@
-# OpenAI Jupyter Notebook Wrapper
+# LMNOP
 
-A simple and intuitive wrapper that allows users to easily interact with the OpenAI API directly within Jupyter Notebooks. This wrapper simplifies the process of querying the OpenAI models and makes it more accessible for researchers, developers, and data scientists working in a notebook environment.
+LMNOP (LLM-based Microbial Metadata Notebook Processing) is a tool presented in the form of a jupyter notebook that allows users to easily clean and summarize metadata fields with support for attributes commonly relevant to microbiome studies, such as summary statistics and visualizations. Users can directly query an LLM within a jupyter notebook in ways that automatically provide context to queries based on the loaded metadata file. With automated improved prompt engineering, obtaining and running code suggestions from ChatGPT. The notebook also provides functions for anomaly detection and column name correcting are presented. Voice recognition is also supported through OpenAI's extremely accurate whisper voice transcription library.
 
-## Key Features
 
-- **Seamless Integration**: Designed specifically for Jupyter Notebooks, ensuring a smooth user experience.
-- **Simplified API Calls**: Reduces boilerplate code, making it easier to query OpenAI models.
-- **Customizable**: Provides flexibility to fine-tune parameters for your specific needs.
-- **Error Handling**: Built-in error handling for common OpenAI API issues.
+## Installation
 
-## Requirements
+To use this notebook, clone the repository and create a new conda environment.
+An environment file will set the name of your env to `lmnop`:
 
-This package requires the following dependencies:
+```bash
+git clone https://github.com/dpear/LMNOP.git
+cd LMNOP
+conda env create -f environment.yml
+```
 
-- `pandas` - For data handling and manipulation.
-- `numpy` - For numerical operations.
-- `openai` - Official OpenAI Python client.
+If you wish to use within an existing conda environment:
+```bash
+conda env update --name existing_env_name --file environment.yml
+```
 
-## Installation
+Then, activate the environment to add a jupyter notebook kernel:
+```bash
+conda activate lmnop
+python -m ipykernel install --user --name lmnop --display-name "Python (lmnop jupyter kernel)"
+```
 
-To use this wrapper, clone the repository and install it directly from GitHub:
+## Using the Notebook
 
+First activate the environment, then actiate the jupyter notebook server (please note that you must activate the environment before starting the jupyter notebook server):
 ```bash
-pip install git+https://github.com/your_username/openai-jupyter-wrapper.git
+conda activate lmnop
+jupyter notebook
+```
+Then navigate to the `proof_of_concept_template_ai_memorializing_DP.ipynb` notebook and run to see an example of the different supported summarization and cleaning functions.
+
```

Comment:
I also note that with qiimp removed, there is no information on how to get one's cleaned metadata back OUT of the notebook. This should probably be added here and/or in the tutorial notebook.

Reply:
added to readme

Reply:
added export function